data/json/recipes/tools/tools_primitive.json

```diff
@@ -335,5 +335,18 @@
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "HAMMER_SOFT", "level": 1 } ],
     "proficiencies": [ { "proficiency": "prof_knapping" }, { "proficiency": "prof_knapping_speed" } ],
     "components": [ [ [ "rock_flaking", 1 ], [ "sharp_rock", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "leather_filing",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "skills_required": [ "survival", 2 ],
+    "time": "5 s",
+    "reversible": true,
+    "autolearn": true,
+    "components": [ [ [ "material_sand", 1 ] ], [ [ "leather", 1 ] ] ]
```

Comment:
I can't find any references to this stuff IRL, how is this made? What makes the sand stick to the leather to be used for filing?

Reply:
Now that you mention it, I think you put a container under whatever your sanding and then put sand on top of it then pull the leather back on forth on it and reuse the sand from the container below.

Reply:
This was my source: https://craftknights.com/10-ways-to-smooth-wood-without-sandpaper/
Nothing really holds the sand as far as I understand the source. You just apply it to the surface being polished as needed and rub with the leather.

Reply:
I see. I guess this is an okay solution.